File: Day_27/Q106.java
import java.util.*;

//Class To Store Employee Details
class Employee{
    int id;
    String name;
    float salary;
}

class Q106{

    //Function To Input Employee Details
    static void input_employee(Employee e,Scanner sc){

        System.out.print("Enter Employee ID : ");
        e.id = sc.nextInt();

        sc.nextLine();

        System.out.print("Enter Employee Name : ");
        e.name = sc.nextLine();

        System.out.print("Enter Salary : ");
        e.salary = sc.nextFloat();
    }

    //Function To Display Employee Details
    static void display_employee(Employee e){

        System.out.println("\nEmployee ID : " + e.id);
        System.out.println("Employee Name : " + e.name);
        System.out.println("Salary : " + e.salary);
    }

    public static void main(String args[]){

        Scanner sc = new Scanner(System.in);

        int n;

        //Input Number Of Employees
        System.out.print("Enter Number Of Employees : ");
        n = sc.nextInt();

        //Array Of Objects
        Employee emp[] = new Employee[n];

        //Input Employee Records
        for(int i = 0;i < n;i++){

            emp[i] = new Employee();

            System.out.println("\nEnter Details Of Employee " + (i + 1));

            input_employee(emp[i],sc);
        }

        //Display Employee Records
        System.out.println("\n----- Employee Records -----");

        for(int i = 0;i < n;i++){
            display_employee(emp[i]);
        }

        sc.close();
    }
}

File: Day_27/Q106.cpp
#include<bits/stdc++.h>
using namespace std;

//Structure To Store Employee Details
struct Employee{
    int id;
    string name;
    float salary;
};

//Function To Input Employee Details
void input_employee(Employee &e){

    cout << "Enter Employee ID : ";
    cin >> e.id;

    cin.ignore();

    cout << "Enter Employee Name : ";
    getline(cin,e.name);

    cout << "Enter Salary : ";
    cin >> e.salary;
}

//Function To Display Employee Details
void display_employee(Employee &e){

    cout << "\nEmployee ID : " << e.id << endl;
    cout << "Employee Name : " << e.name << endl;
    cout << "Salary : " << e.salary << endl;
}

int main(){

    int n;

    //Input Number Of Employees
    cout << "Enter Number Of Employees : ";
    cin >> n;

    //Vector Of Structure
    vector<Employee> emp(n);

    //Input Employee Records
    for(int i = 0;i < n;i++){

        cout << "\nEnter Details Of Employee " << i + 1 << endl;

        input_employee(emp[i]);
    }

    //Display Employee Records
    cout << "\n----- Employee Records -----\n";

    for(int i = 0;i < n;i++){
        display_employee(emp[i]);
    }

    return 0;
}